frontend/src/components/maps/leaflet/InfoSlideOut/AssociatedBuildingsList.tsx

```diff
@@ -0,0 +1,67 @@
+import { IParcel } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import * as React from 'react';
+import { ReactElement } from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+
+interface IAssociatedBuildings {
+  /** the selected property information */
+  propertyInfo: IParcel | null;
+  /** link that opens the sidebar to create a new building associated to the parcel */
+  addAssociatedBuildingLink: ReactElement;
+  /** whether the user has the correct agency/permissions to edit property details */
+  canEditDetails: boolean;
+}
+
+/**
+ * Component that displays the associated buildings of a parcel
+ * as a clickable list
+ * @param propertyInfo the selected parcel
+ * @param addAssociatedBuildingLink link to create a new associated building
+ * @param canEditDetails whether the user can edit the parcel details
+ */
+export const AssociatedBuildingsList: React.FC<IAssociatedBuildings> = ({
+  propertyInfo,
+  addAssociatedBuildingLink,
+  canEditDetails,
+}) => {
+  const location = useLocation();
+  return (
+    <>
+      <ListGroup>
+        <Label className="header" style={{ margin: '5px 0px' }}>
+          Associated Buildings
+        </Label>
+        <ListGroup.Item>Click a building name to view its details</ListGroup.Item>
+        {propertyInfo?.buildings.map((building, buildingId) => (
```

Comment:
The second param is the index of the building within the buildings array, not the buildingId there is no guarantee that the index will be the same as the buildingId.

Reply:
should I just use building.id instead? I currently use it for the "to" link